cmd/kops/rollingupdatecluster.go

```diff
@@ -44,6 +44,8 @@ type RollingUpdateOptions struct {
 	NodeInterval    time.Duration
 	BastionInterval time.Duration
 
+	Headroom int64
```

Comment:
I wonder if we should borrow the deployment logic and/or terminology.  Is this the same as maxSurge?

https://kubernetes.io/docs/user-guide/deployments/#rolling-update-deployment

(Not saying we should copy and paste the code - unless someone wants to show that is actually easy - but just saying that we should head in that direction by using the familiar terminology)